### PR TITLE
scrapy 2.9.0 (new formula)

### PR DESCRIPTION
**Pull request by p-linnane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/scrapy.rb [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  homepage "https://scrapy.org"
+  url "https://files.pythonhosted.org/packages/eb/b0/7da031ff0ce073839984097724db2c54e8c4b7938acb9ef4691a98620ce7/Scrapy-2.9.0.tar.gz"
+  sha256 "564c972b56e54b83141f395ce3f6a25bfe2093d61d13f9b81d05384e19db98da"
+  license "BSD-3-Clause"
+  head "https://github.com/scrapy/scrapy.git", branch: "master"
+
+  # `pkg-config`, `rust`, and `openssl@1.1` are for cryptography.
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "openssl@1.1"
```
Comment (carlocab) [inline Formula/scrapy.rb:14]: Let's wait until after the OpenSSL migration for this, I think.

Comment (p-linnane) [reply Formula/scrapy.rb:14]: Yes, just running it through CI to see if there are any other issues on older macOS.

Comment (p-linnane) [reply Formula/scrapy.rb:14]: That's a better idea I didn't think of. I'll open a PR targetting staging.

Comment (p-linnane) [reply Formula/scrapy.rb:14]: #134777

**Pull request closed without merging**